.github/workflows/build-timestamped-master.yml

```diff
@@ -4,6 +4,9 @@ on:
     push:
         branches:
             - master
+        paths-ignore:
+            - '**/.md'
```

Comment:
This will ignore Module.md changes too, won't it? Would be an issue if we have to check any doc changes

Reply:
Good point. We may have to include them separately. Will update the PR.

Reply:
Fixed with d881a21021909eb442dc765d441ba5be7c0bccca